encoding/thrift/benchmark_test.go

```diff
@@ -0,0 +1,49 @@
+package thrift
+
+import (
+	"bytes"
+	"context"
+	"testing"
+	"time"
+
+	"go.uber.org/yarpc/api/transport"
+	"go.uber.org/yarpc/api/transport/transporttest"
+
+	"github.com/golang/mock/gomock"
+	"go.uber.org/thriftrw/wire"
+)
+
+func BenchmarkThrift(b *testing.B) {
```

Comment:
are you bench-marking the initialization of the test? if not, testing.B offers a way to reset the timer before running the b.N loop.